plugins/grpc/java/v1.63.0/buf.plugin.yaml

```diff
@@ -0,0 +1,25 @@
+version: v1
+name: buf.build/grpc/java
+plugin_version: v1.63.0
+source_url: https://github.com/grpc/grpc-java
+integration_guide_url: https://grpc.io/docs/languages/java/quickstart
+description: Generates Java client and server stubs for the gRPC framework.
+deps:
+  - plugin: buf.build/protocolbuffers/java:v25.3
```

Comment:
We need to keep this pinned to < 26.x until the upstream https://github.com/grpc/grpc-java/issues/11015 issue is resolved.